dispatch/dispatch_types.go

```diff
@@ -30,7 +30,7 @@ type Dispatcher struct {
 	// See relayer() method below.
 	routes map[uuid.UUID][]chan interface{}
 	// rMtx protects the routes variable ensuring acceptable read/write access
-	rMtx sync.RWMutex
+	rMtx sync.Mutex
```

Comment:
Very petty, but
```suggestion
mtx sync.Mutex
```

Reply:
Not petty at all.
Words mean things.

However it wasn't named rMtx as in ReadMutex.
It's named after Routes. Not that I named it, but still, I'm pretty confident.

Fixed 545fa0cb

Reply:
Oh I think you're correct and that I misread it (by looking at changes vs reading the comment 👀). I like what you've named it 👍